MapboxDirections/MBDirections.swift

```diff
@@ -121,6 +121,9 @@ open class Directions: NSObject {
     /// The Mapbox access token to associate the request with.
     internal let accessToken: String
     
+    /// The routeIdentifier for the request.
+    internal var routeIdentifier: String?
```

Comment:
A single Directions object can produce multiple requests that may come back in a different order. Storing this information on Directions makes the class vulnerable to race conditions.